src/utils/equipment-types.js

```diff
@@ -13,6 +13,20 @@ const Countries = {
         type: filteredTypes.countries,
     },
 };
+const EnergySource = {
+    energySource: {
+        name: 'EnergySourceText',
+        type: filteredTypes.enum,
+        enumValues: [
+            { id: 'HYDRO', label: 'Hydro' },
```

Comment:
I would favor the name 'i18n_id' over 'label', as the latter is not a real label, as it as to undergo the internationalisation step.
I would even simplify the mapping aspect, by relying on js object syntax and mecanisms :
```
enumValues : {
  HYDRO : 'Hydro', NUCLEAR: 'Nuclear', // and so on
}
```
that would save you the burden of naming the parts.